### PR TITLE
Added functionality to render markdown

### PR DESCRIPTION
**Pull request by arjunmahishi**
Addressing #1

### CODE REVIEW

File: ghp.go [hunk 1]
```diff
@@ -55,3 +56,40 @@ func main() {
 	http.HandleFunc("/", handler)
 	log.Fatal(http.ListenAndServe(fmt.Sprintf(":%d", port), nil))
 }
+
+func renderFile(w http.ResponseWriter, r *http.Request, filename string) {
+	nameSlice := strings.Split(filename, ".")
```
Comment (CurtisLusmore) [inline ghp.go]: Consider using
```suggestion
	ext:= path.Ext(filename)
```
and adding `"path"` to the import statement at the top.

**Review by CurtisLusmore — APPROVED** [review]
LGTM

**Comment by CurtisLusmore** [comment]
Awesome, thank you so much for contributing this change @arjunmahishi !

**Comment by arjunmahishi** [comment]
Thank you! :)

**Pull request merged**